### PR TITLE
adding wildcard example script

### PR DESCRIPTION
**Pull request by jking-netuitive**

### CODE REVIEW

**Comment by TheConnMan** [comment]
Does this belong in https://github.com/Netuitive/zorka-custom-mbean-examples? Should that repo be included in this project instead?

File: zorka-core-test/src/main/resources/scripts/wildcard-example.bsh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// This is sample script intended to help understanding how to implement arbitrary method instrumentation.
+
+// To run:
+//  1. update zorka.properties to add it to list of loaded scripts, i.e. scripts = jvm.bsh, wildcard-example.bsh
+//  2. update zorka.properties to set zorka.application = Caculator
```
Comment (Grumblesmith) [inline zorka-core-test/src/main/resources/scripts/wildcard-example.bsh]: I recommend spelling this "Calculator"

File: zorka-core-test/src/main/resources/scripts/wildcard-example.bsh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+//  3. get the api key by creating a java datasource on Netuitive Cloud: [https://app.netuitive.com].
+//  4. update zorka.properties to set netuitive.api.key using the key obtained from previous step.
+//  5. assuming 'java' command is available from system path, type 'java -javaagent:./netuitive.jar=. -jar zorka-core-test.jar
+//  6. upon prompt, start typing 1+2[enter] 3-3[enter] 2*3[enter] 3/3[enter] 3/0[enter] ...
+//  7. observe a jvm element named 'Calculator' shows up in Netuitive with genernal jvm system metrics and the CalculatorStats method call metrics
```
Comment (Grumblesmith) [inline zorka-core-test/src/main/resources/scripts/wildcard-example.bsh]: general

File: zorka-core-test/src/main/resources/scripts/wildcard-example.bsh (hunk 1 above)
Comment (jking-netuitive) [reply zorka-core-test/src/main/resources/scripts/wildcard-example.bsh]: I'd like to point out that this is copy pasted from calculator.bsh, of which I am not the author haha

Comment (TheConnMan) [reply zorka-core-test/src/main/resources/scripts/wildcard-example.bsh]: If you copy paste it you should still look it over.

File: zorka-core-test/src/main/resources/scripts/wildcard-example.bsh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        //adding them to the spy map
+        record.put("METHOD_NAME", methodName);
+        record.put("CLASS_NAME", packageName == null ? packagePath : packageName);
+    }
+    return record;
+  }
+  return this;
+} // __mbsRegister()
+
+// BeanShall has no notion of namespaces, so we simulate namespace using function
```
Comment (Grumblesmith) [inline zorka-core-test/src/main/resources/scripts/wildcard-example.bsh:43]: Beanshell

**Comment by TheConnMan** [comment]
This script should live in the examples repo.

**Pull request closed without merging**